src/vm/amd64/jithelpers_fast.S

```diff
@@ -110,7 +110,7 @@ LEAF_ENTRY JIT_WriteBarrier, _TEXT
         movabs  rax, 0xF0F0F0F0F0F0F0F0
 
         // Touch the card table entry, if not already dirty.
-        shr     rdi, 0Bh
+        shr     rdi, 0x0B
```

Comment:
Thanks a lot! I wonder if it is a bug or a feature :-). Anyways, it is good to stay consistent with hex format usage in the asm files, so this change is a good thing no matter what.